index.html

```diff
@@ -8,7 +8,7 @@
  <body>
    
   <section class="hero">
-    <div class="hero-inner">
+    <div class="hero-inner" align="center">
```

Comment:
while it will work, it is hard to maintain, we have styling in different parts of the code, and you don't have the ability to trigger changes with media-queries, for example you may not want to be text aligned center on mobiles. 

Use you favorite solution, you were trying to solve previous problem with alignment block in the center